packages/web-components/src/components/va-button/va-button.css

```diff
@@ -20,29 +20,29 @@ button:hover {
   background-color: var(--color-primary-darker);
 }
 
-:host([back]) button,
-:host([secondary]) button {
+:host([back]:not([back='false'])) button,
+:host([secondary]:not([secondary='false'])) button {
```

Comment:
The `:not([prop='false'])` changes were done because Storybook does not correctly omit boolean props when false. Storybook incorrectly sets the boolean prop to `="false"` in the DOM.